src/getHardhatConfigNetworks/index.test.ts

```
@@ -0,0 +1,174 @@
+    }).toThrow('PRIVATE_KEY env variable is not a valid private key');
+  });
+
+  it('should use EVM API URL when available', () => {
+    const result = typedGetHardhatConfigNetworks();
+
+    // Both test networks have EVM APIs, so they should be selected
+    expect(result.eth_mainnet.url).toBe('https://eth.drpc.org'); // EVM API
+    expect(result.bsc_testnet.url).toBe('https://bsc-testnet.drpc.org'); // EVM API
+  });
+
+  it('should return empty string for URL when no EVM API is available', () => {
```

ESLint

[error] Replace `'should·return·empty·string·for·URL·when·no·EVM·API·is·available'` with `"should·return·empty·string·for·URL·when·no·EVM·API·is·available"`

(prettier/prettier)